src/scss/variables.scss

```diff
@@ -0,0 +1 @@
+$primary: #053b8d;
```

Comment:
The name of a partial file should start with an underscore(_)

Reply:
I have renamed the partial file: variables.scss to _variables.scss